packages/react-examples/src/react-components/Migrations/utils.tsx

```diff
@@ -45,7 +45,7 @@ export const CodeExample = (props: { title?: string; children: React.ReactElemen
   const language = (value.includes('```')
     ? value.substring(3, value.indexOf('\n'))
     : 'jsx') as keyof typeof codeLanguages;
-  const code = value.substring(value.indexOf('\n')).replace('```', '').trim();
+  const code = value.replace(`\`\`\`${language}`, '').replace('```', '').trim();
```

Comment:
**suggestion:** whilst string literals are preferable this makes it a bit unreadable IMO.

I'd suggest to go with standard `+` concatenation for sake of readability 
->

```
const codeBlockTag = '```'
value
  .replace(codeBlockTag + language, '')
  .replace(codeBlockTag, '')
  .trim()
```